.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,111 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
nie powinniśmy dodawać plików z katalogu .idea do repo (to są ustawienia IDE Twojego środowiska)
w pliku .gitignore powinna znaleźć się taka linijka ".idea" ale teraz musiałabyś usunąć ten katalog ze śledzenia gita. Piszę o tym abyś wiedziała o tym na przyszłość.